### ISSUE TITLE
Editor: Unable to add any new block between 2 Koenig blocks

### ISSUE THREAD

**Issue by fadeevab** [TryGhost/Ghost#10659]
### Issue Summary
Impossible to add a new Markdown, image of HTML block between the newly created "rich" paragraphs in default Ghost editor (Koenig?).

### To Reproduce

I don't know: sometimes the block is just removed only once (so you can add afterwards), but sometimes **it's never possible to add a new markdown/image/html block**.

### Technical details:

* Ghost Version: Ghost 2.19.2 
* Node Version: 8.15.1
* Browser/OS: Firefox 66.0.2
* Database: MySQL

**Comment by fadeevab** [comment]
It happens not always. Sometimes the markdown becomes visible and it is removed immediately in a second automatically. Sometimes need to play with text and so on.

**Comment by fadeevab** [comment]
OK, need to re-formulate the issue.

**Comment by kevinansfield** [comment]
@fadeevab it's very difficult to follow this issue and I'm not able to replicate the behaviour I _think_ you are describing. Can you please share a screen recording that demonstrates the problem? Also any error messages from the web inspector console if you get them.

**Comment by fadeevab** [comment]
Yes, I will add more info one I reproduce again, and I created another issue about the loosing of some edits because of auto-saving.

**Comment by fadeevab** [comment]
I'm going to close the issue if it's not reproduced in a few days while I'm creating a new blog post.

**Comment by fadeevab** [comment]
Please, take a look at #10661.
